src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Avoid using the universal selector '*' for setting styles. It impacts performance as it applies styles to all elements in the DOM. Set styles only for elements that require them.